src/applications/vaos/appointment-list/redux/selectors.js

```diff
@@ -185,190 +176,13 @@ export const selectPastAppointmentsV2 = createSelector(
     }
 
     const sortedAppointments = past
-      .filter(isValidPastAppointmentOrExpressCare)
+      .filter(isValidPastAppointment)
```

Comment:
Do we need to filter for express care here as well since `isValidPastAppointment` doesn't have that check?

Reply:
🤦 your right

Reply:
Yeah, I think for most of these selectors, I think we want to put checks to exclude Express Care in the filter functions, rather than putting them in these selectors themselves. Looks like `isUpcomingAppointmentOrRequest`, `isCanceledConfirmed`, `isValidPastAppointment`, and `isUpcomingAppointment` would be the ones to update.

It's possible that there's an issue in the filtering somewhere that's leading to this bug on the homepage:

![Screen Shot 2021-05-12 at 9 42 18 AM](https://user-images.githubusercontent.com/634932/117985035-5c05ed00-b306-11eb-85ed-3352537f33c3.png)